eve/workers/end2end/scripts/configure-e2e.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+set -exu
+
+ZENKO_NAME=${1:-end2end}
+NAMESPACE=${2:-default}
+
+SERVICE_ACCOUNT="${ZENKO_NAME}-config"
+POD_NAME="${ZENKO_NAME}-config"
+MANAGEMENT_ENDPOINT="http://${ZENKO_NAME}-management-orbit-api:5001"
+UUID=$(kubectl get zenko ${ZENKO_NAME} --namespace ${NAMESPACE} -o jsonpath='{.status.instanceID}')
+KEY=$(kubectl get secret end2end-management-orbit-api-jwt-secret.v1 --namespace ${NAMESPACE} -o jsonpath='{.data.jwtUserSecret}' | base64 -d)
+
+cat <<EOF | kubectl apply -f - 
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ${SERVICE_ACCOUNT}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ${SERVICE_ACCOUNT}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - '*'
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ${SERVICE_ACCOUNT}
+subjects:
+- kind: ServiceAccount
+  name: ${SERVICE_ACCOUNT}
+roleRef:
+  kind: Role
+  name: ${SERVICE_ACCOUNT}
+  apiGroup: rbac.authorization.k8s.io
+EOF
+
+kubectl run ${POD_NAME} \
+  --image "registry.scality.com/zenko-dev/zenko-e2e:latest" \
```

Comment:
let's avoid using `zenko-dev` as I'm planning to deprecate it (this subject remains to be discussed approved together)
```suggestion
  --image "registry.scality.com/zenko/zenko-e2e:latest" \
```

Reply:
at the moment, the end2end image is only built and loaded directly into kind, and it is not pushed to the repo.
question: should there be an image repo for end2end images?

Reply:
I don't mind that we add a steps/stage that push this images into `registry.scality.com/zenko/zenko-e2e` could be nice to have this image here if we on our computers or cluster wants to pull it and launch some tests.

But that can be done in a later step I believe.